api/tests/core/users/young_status_test.py

```diff
@@ -13,19 +13,19 @@
 pytestmark = pytest.mark.usefixtures("db_session")
 
 
+def withAge(age):
```

Comment:
<!-- Chores are simple tasks that must be done before the subject can be “officially” accepted. Usually, these comments reference some common process. Try to leave a link to the process description so that the reader knows how to resolve the chore.  -->
**💣 chore:** ‏une fonction locale comme celle-ci on la nomme conventionnellement `_with_age`